src/viewer/controls.styl

```diff
@@ -24,6 +26,9 @@
             margin-left 20%
 
 .pho-viewer-nav
+    position   fixed
+    top        5.625rem // because of the toolbar
```

Comment:
Is it possible to get this height via a variable? Using it to set this position and the height of the toolbar?